tests/lib/args_validator.py

```diff
@@ -194,6 +194,71 @@ def test_ValidateFailure(self):
         self.validator.Validate(value, params)
 
 
+class DatetimeValidatorTest(unittest.TestCase):
```

Comment:
Could we also add a test for the variable interpolation here? i.e. that specifying two args that depend on one another (like in your example) works / fails as intended

Reply:
Raised 713. I don't think the test should go here though, but where that interpolation is performed, since by the time the values make it to DatetimeValidator, there's no indication whether they were interpolated or not.